ruby/vampires.rb

```diff
@@ -5,7 +5,7 @@
 
 while counter <= numbr.to_i

```

Comment:
If we have a number that is our specific break point for a loop, check out some other loops that might fit the job better, especially `x.times do`